common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -292,7 +292,7 @@ trait FeatureSwitches {
     "ipad-split-capabilities",
     "If switched on then this gives older ipads the stripped down front but full articles",
     safeState = On,
-    sellByDate = new LocalDate(2015, 9, 30),
+    sellByDate = new LocalDate(2015, 11, 30),
```

Comment:
November 30th is a Monday. Will this revert effectively on Sunday night? Do we care about that?